pkg/arvo/app/btc-wallet.hoon

```diff
@@ -125,73 +126,536 @@
   ==
 ::
 ++  on-poke
+  ~/  %on-poke
   |=  [=mark =vase]
   ^-  (quip card _this)
+  |^
   =^  cards  state
-  ?+  mark  (on-poke:def mark vase)
-      %btc-wallet-command
-    ?>  =(our.bowl src.bowl)
-    (handle-command:hc !<(command vase))
-    ::
-      %btc-wallet-action
-    ?<  =(our.bowl src.bowl)
-    (handle-action:hc !<(action vase))
-    ::
-      %btc-wallet-internal
-    ?>  =(our.bowl src.bowl)
-    (handle-internal:hc !<(internal vase))
-  ==
+    ?+  mark  (on-poke:def mark vase)
```

Comment:
Large transfers of code between the cores make me nervous, can be hard to detect a missing `:hc`. Needs to be tested thoroughly.

Reply:
Yes absolutely. For the record I've gone through and manually searched every arm call to ensure the hc is there where appropriate. I'm currently running this on my ship and it's working good. What would you like me to test?

Reply:
I mean, basically all of it. Delete provider, set provider again, delete wallet, set wallet again, send btc to a ship, send btc to a non-ship address, receive btc from a ship, receive btc from a non-ship address

Reply:
happy to show you how to set up testnet if you want to do it that way.

Reply:
Yeah, can we spin up a testnet tomorrow and go through it?

Reply:
Sure